test/tests/components/ws-manager/tasks_test.go

```diff
@@ -82,37 +79,6 @@ func TestRegularWorkspaceTasks(t *testing.T) {
 						_ = integration.DeleteWorkspace(ctx, api, nfo.Req.Id)
 					})
 
-					conn, err := api.Supervisor(nfo.Req.Id)
```

Comment:
I'd prefer we comment out these line and add TODO, then we would not forget to adding these test back in the future. WDYT?

Reply:
I have edited original issue to break down tasks. We don't need to add a todo here.